### PR TITLE
T5 Model Cards

### LINKED ISSUE #5727: t5 model card
Add Model Card for all t5 checkpoints.
cc @clmnt

### PR DESCRIPTION
**Pull request by sshleifer**
All are identical.
Happy to update w more info!
I didn't have task tags because they seem to already work.

### CODE REVIEW

File: model_cards/t5-11b/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+![model image](https://camo.githubusercontent.com/623b4dea0b653f2ad3f36c71ebfe749a677ac0a1/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f343030362f312a44304a31674e51663876727255704b657944387750412e706e67)
+
```
Comment (julien-c) [inline model_cards/t5-11b/README.md:23]: ^^ Will this link work forever?

File: model_cards/t5-11b/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+--
```
Comment (julien-c) [inline model_cards/t5-11b/README.md:1]: you need exactly three `---`, won't work otherwise

File: model_cards/t5-11b/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+language: english
```
Comment (julien-c) [inline model_cards/t5-11b/README.md:2]: en

**Comment by julien-c** [comment]
The file paths aren't correct. (we have a "legacy" format for non-namespaced models – model pages link to the correct paths)

In fact if you look at the model pages, e.g. https://huggingface.co/t5-base – they already have (non-textual) model cards. Can you update those? Thanks=)

**Comment by julien-c** [comment]
Up, @sshleifer

File: model_cards/t5-11b/README.md (hunk 1 above)
Comment (sshleifer) [reply model_cards/t5-11b/README.md]: How can I determine that? It's from @mrm8488 's model card.

**Comment by sshleifer** [comment]
CircleCI jobs say they succeeded if you click through. Unclear why they are yellow on this page.

**Pull request merged**